src/job-cache/job-cache.cpp

```diff
@@ -86,35 +87,30 @@ static void rmdir_no_fail(const char *dir) {
 // supported by the filesystem it copies instead.
 #ifdef __APPLE__
 
```

Comment:
I'd have assumed that the shared-cache were linux-only, as the macos implementation of wakebox can't hide the .fuse directory when a payload command resolves an absolute path

Reply:
The shared cache itself should be able to work on mac but that may be an issue that I was about to spend a lot time debugging. The tests failed when they ran on mac despite succeeding on the tarball step. I guess I need to update these tests to pass without testing on mac.

Reply:
Thanks for the heads up!